isis/version

```diff
@@ -1,5 +1,2 @@
-3.5.3.0      # Public version number
-2018-08-01   # Version date
```

Comment:
Do we want the version date any more? If we don't know what the date is going to be we can always put the month and year rather than the actual day.